Source/Core/DoublyLinkedList.js

```diff
@@ -0,0 +1,105 @@
+/*global define*/
+define([
+        '../Core/defined',
+        '../Core/defineProperties'
+    ], function(
+        defined,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function DoublyLinkedList() {
```

Comment:
Sometimes we provide reference doc for `@private` types, sometimes we do not.  It is not a requirement and sometimes a benefit of `private` is that we don't have to spend the time writing the doc.  Let's pass on this for this PR.